spinn_front_end_common/interface/provenance/provides_provenance_data_from_machine_impl.py

```diff
@@ -284,4 +286,9 @@ def get_provenance_data_from_machine(self, transceiver, placement):
         """
         provenance_data = self._read_provenance_data(
             transceiver, placement)
-        return self._read_basic_provenance_items(provenance_data, placement)
+        items = self._read_basic_provenance_items(provenance_data, placement)
+        label, x, y, p, names = self._get_placement_details(placement)
```

Comment:
Why is self._get_placement_details(placement) done here AND in self._read_basic_provenance_items( which is only called from here?

Why not have read_basic_provenance_item and get_remaining_provenance_data_items take the same parameters?

Reply:
That puzzled me too.